### PR TITLE
Added patch route in backend to fix share recipe count

### LINKED ISSUE #390: BUG: Recipe sharing feature does not work
### Is there an existing issue raised for this?

- [X] I have searched the existing issues

### Describe your issue

Recipe sharing button does not work

### Steps to reproduce

Go to recipe -> Hit share button

### What was the expected result?

No error

### Proposed solution

Fix frontend onClick element
Animation4

### Would you like to work on this issue?

Yes

### Terms & Conditions

- [X] I agree to follow the Code of Conduct
- [X] I'm a GSSOC'24 Ext contributor

### PR DESCRIPTION
**Pull request by mehul-m-prajapati**
- Added patch route in backend to fix share recipe count
Fixes #390 

Animation1

## How should this be tested?
- Hit share button of recipe

### Required

- [x] Filled out the "How to test" section in this PR
- [x] Read Contributing guide
- [x] Self-reviewed my own code
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] The changes don't cause any responsiveness issues

### CODE REVIEW

File: frontend/src/Components/Cards.jsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ const Cards = ({ dish, setRecipes, recipes, index }) => {
 
         setRecipes(updatedRecipes);
         await navigator.clipboard.writeText(postURL);
-        toast.success("Copied!");
+        toast.success("URL copied to clipboard successfully");
       } else {
```
Comment (AlfiyaSiddique) [inline frontend/src/Components/Cards.jsx:60]: @mehul-m-prajapati  Keep the text as it was before!

**Comment by AlfiyaSiddique** [comment]
@mehul-m-prajapati one comment rest looks good to me

File: frontend/src/Components/Cards.jsx (hunk 1 above)
Comment (mehul-m-prajapati) [reply frontend/src/Components/Cards.jsx]: @AlfiyaSiddique : okay updated it

**Comment by mehul-m-prajapati** [comment]
@AlfiyaSiddique : okay thanks for review.

**Pull request merged**